Library/ViewModels/PledgeSummaryCellViewModel.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+import Prelude
+import ReactiveSwift
+
+public protocol PledgeSummaryCellViewModelInputs {
+  func tapped(_ url: URL)
```

Comment:
In other places where `tapped`  is used, we usually make the argument name explicit. It's easier to read and understand what was tapped, so by using `tapped(url: URL)`, we call it with `tapped(url: $0)`, instead of `tapped($0)`, for example.

Reply:
I just noticed that this code is from the #717 😛 . But feel free to keep how it is if you think it's not necessary.

Reply:
As long as the type and be inferred from the function signature, in this case `URL` we don't need to reiterate that in the argument name. If this was `tapped(index: Int)` we would keep `index` as an argument because it's not obvious from the type what `index` is. Makes sense?